CLI/commands/sto_manager.js

```diff
@@ -283,53 +284,110 @@ function fundingConfigUSDTieredSTO() {
-    funding.raiseType.push(gbl.constants.FUND_RAISE_TYPES.DAI);
+  if (selectedFunding.includes('S')) {
+    funding.raiseType.push(gbl.constants.FUND_RAISE_TYPES.STABLE);
   }
   if (funding.raiseType.length == 0) {
-    funding.raiseType = [gbl.constants.FUND_RAISE_TYPES.ETH, gbl.constants.FUND_RAISE_TYPES.POLY, gbl.constants.FUND_RAISE_TYPES.DAI];
+    funding.raiseType = [gbl.constants.FUND_RAISE_TYPES.ETH, gbl.constants.FUND_RAISE_TYPES.POLY, gbl.constants.FUND_RAISE_TYPES.STABLE];
   }
 
   return funding;
 }
 
-function addressesConfigUSDTieredSTO(usdTokenRaise) {
-  let addresses = {};
+async function addressesConfigUSDTieredSTO(usdTokenRaise) {
 
-  addresses.wallet = readlineSync.question('Enter the address that will receive the funds from the STO (' + Issuer.address + '): ', {
-    limit: function (input) {
-      return web3.utils.isAddress(input);
-    },
-    limitMessage: "Must be a valid address",
-    defaultInput: Issuer.address
-  });
-  if (addresses.wallet == "") addresses.wallet = Issuer.address;
+  let addresses, menu;
 
-  addresses.reserveWallet = readlineSync.question('Enter the address that will receive remaining tokens in the case the cap is not met (' + Issuer.address + '): ', {
-    limit: function (input) {
-      return web3.utils.isAddress(input);
-    },
-    limitMessage: "Must be a valid address",
-    defaultInput: Issuer.address
-  });
-  if (addresses.reserveWallet == "") addresses.reserveWallet = Issuer.address;
+  do {
+
+    addresses = {};
 
-  if (usdTokenRaise) {
-    addresses.usdToken = readlineSync.question('Enter the address of the USD Token or stable coin (' + usdToken.options.address + '): ', {
+    addresses.wallet = readlineSync.question('Enter the address that will receive the funds from the STO (' + Issuer.address + '): ', {
       limit: function (input) {
         return web3.utils.isAddress(input);
       },
       limitMessage: "Must be a valid address",
-      defaultInput: usdToken.options.address
+      defaultInput: Issuer.address
     });
-    if (addresses.usdToken == "") addresses.usdToken = usdToken.options.address;
-  } else {
-    addresses.usdToken = '0x0000000000000000000000000000000000000000';
-  }
+    if (addresses.wallet == "") addresses.wallet = Issuer.address;
+  
+    addresses.reserveWallet = readlineSync.question('Enter the address that will receive remaining tokens in the case the cap is not met (' + Issuer.address + '): ', {
+      limit: function (input) {
+        return web3.utils.isAddress(input);
+      },
+      limitMessage: "Must be a valid address",
+      defaultInput: Issuer.address
+    });
+    if (addresses.reserveWallet == "") addresses.reserveWallet = Issuer.address;
+  
+    let listOfAddress;
+  
+    if (usdTokenRaise) {
+      addresses.usdToken = readlineSync.question('Enter the address (or many addresses that you want separated by comma) of the USD Token or stable coin (' + usdToken.options.address + '): ', {
```

Comment:
I was just read this and it didn't sound correct so this is my suggested edit. Also "Stable coin" is used throughout and not "USD Token" so suggest removing it. No harm saying "USD stable coin" explicitly in case someone tries to use a different currency stable coin and as the Oracles are USD only. 
```suggestion
      addresses.usdToken = readlineSync.question('Enter the address (or multiple addresses separated by commas) of the USD stable coin(s) (' + usdToken.options.address + '): ', {
```

Reply:
Thanks @F-OBrien, you have right!!! I will modify the string in the next commit.

Reply:
@shuffledex I have merged the suggestion, please pull on your local.